### PR TITLE
Performance benchmark update in readme

### PR DESCRIPTION
**Pull request by AnishaUdayakumar**

### CODE REVIEW

File: recipes/intelligent_queue_management/README.md [hunk 1]
```diff
@@ -104,4 +104,33 @@ _NOTE: Alternatively, you can run all steps with the following command:_
...
 
-Congratulations! You have successfully set up and run the Intelligent Queue Management application with OpenVINO™.
+## Step 4
+
+Benchmark the Model with OpenVINO's Benchmark_App
+Benchmarking provides insight into your YOLOv8 model's real-world performance. Performance may vary based on use and configuration.
+
+### Benchmark Results 
+
+![YOLOv8m Benchmark Results](https://github.com/AnishaUdayakumar/intelligent-queue-management-openvino/assets/109281183/8a81243e-ee32-4b30-9994-326ecb07d32f)
```
Comment (adrianboguszewski) [inline recipes/intelligent_queue_management/README.md:114]: These images must belong to the OpenVINO repository (if possible) and should be separate from each other.

File: recipes/intelligent_queue_management/README.md [hunk 2]
```diff
@@ -104,4 +104,33 @@ _NOTE: Alternatively, you can run all steps with the following command:_
 python main.py --stream sample_video.mp4
 ```
 
-Congratulations! You have successfully set up and run the Intelligent Queue Management application with OpenVINO™.
+## Step 4
+
+Benchmark the Model with OpenVINO's Benchmark_App
+Benchmarking provides insight into your YOLOv8 model's real-world performance. Performance may vary based on use and configuration.
```
Comment (adrianboguszewski) [inline recipes/intelligent_queue_management/README.md:110]: These two lines are merged into one paragraph. Was it your assumption?

File: recipes/intelligent_queue_management/README.md [hunk 3]
```diff
@@ -104,4 +104,33 @@ _NOTE: Alternatively, you can run all steps with the following command:_
...
+![YOLOv8m Benchmark Results](https://github.com/AnishaUdayakumar/intelligent-queue-management-openvino/assets/109281183/8a81243e-ee32-4b30-9994-326ecb07d32f)
+
+The benchmarks for the YOLOv8m model were run using OpenVINO version 2023.0.0 on the 4th Gen Intel® Xeon® Scalable Processor:
+
+* **Throughput** 
+  * FP16: 252FPS
+  * INT8: 195 FPS
+* **Latency**
+  * FP16: 7.94 ms
+  * INT8: 10.19 ms
```
Comment (adrianboguszewski) [inline recipes/intelligent_queue_management/README.md:123]: Is this depicted on the images above? If so, it's not needed here

File: recipes/intelligent_queue_management/README.md [hunk 4]
```diff
@@ -104,4 +104,33 @@ _NOTE: Alternatively, you can run all steps with the following command:_
...
+### Running the Benchmark
+
+Use the following command to run the benchmark:
+
+```shell
+!benchmark_app -m $int8_model_det_path -d $device -hint latency -t 30
+```
+Replace `int8_model_det_path` with the path to your INT8 model and $device with the specific device you're using (CPU, GPU, etc.). This command performs inference on the model for 30 seconds. Run `benchmark_app --help` for additional command-line options.
+
+Congratulations! You have successfully set up and run the Intelligent Queue Management application with OpenVINO™ and benchmarked the model.
```
Comment (adrianboguszewski) [inline recipes/intelligent_queue_management/README.md:136]:
```suggestion
Congratulations! You have successfully set up, run, and benchmarked the Intelligent Queue Management application with OpenVINO™.
```

File: recipes/intelligent_queue_management/README.md [hunk 5]
```diff
@@ -104,4 +104,33 @@ _NOTE: Alternatively, you can run all steps with the following command:_
...
+## Step 4
+
+Benchmark the Model with OpenVINO's Benchmark_App
+Benchmarking provides insight into your YOLOv8 model's real-world performance. Performance may vary based on use and configuration.
+
+### Benchmark Results 
+
+![YOLOv8m Benchmark Results](https://github.com/AnishaUdayakumar/intelligent-queue-management-openvino/assets/109281183/8a81243e-ee32-4b30-9994-326ecb07d32f)
+
+The benchmarks for the YOLOv8m model were run using OpenVINO version 2023.0.0 on the 4th Gen Intel® Xeon® Scalable Processor:
```
Comment (adrianboguszewski) [inline recipes/intelligent_queue_management/README.md:116]: The configuration is missing for benchmarking and this is what is required by the legal team.

**Review by adrianboguszewski — CHANGES_REQUESTED** [review]

File: recipes/intelligent_queue_management/README.md (hunk 1 above)
Comment (AnishaUdayakumar) [reply recipes/intelligent_queue_management/README.md:114]: I will add the images separately to the OpenVINO repository. Will work on this change.
Can you help me with some leads to add the data to the OpenVINO repository?

File: recipes/intelligent_queue_management/README.md (hunk 2 above)
Comment (AnishaUdayakumar) [reply recipes/intelligent_queue_management/README.md:110]: > These two lines are merged into one paragraph. Was it your assumption?

Thank you for pointing that out. Yes, I intended to have them as two separate paragraphs and it is my assumption.

File: recipes/intelligent_queue_management/README.md (hunk 5 above)
Comment (AnishaUdayakumar) [reply recipes/intelligent_queue_management/README.md:116]: I have updated the statement with configuration details. Please let me know if any further changes are needed.

File: recipes/intelligent_queue_management/README.md (hunk 3 above)
Comment (AnishaUdayakumar) [reply recipes/intelligent_queue_management/README.md:123]: Yes, these numbers are from the chart. I have removed this content as requested.

File: recipes/intelligent_queue_management/README.md [hunk 6]
```diff
@@ -104,4 +104,26 @@ _NOTE: Alternatively, you can run all steps with the following command:_
 python main.py --stream sample_video.mp4
 ```
 
-Congratulations! You have successfully set up and run the Intelligent Queue Management application with OpenVINO™.
+## Step 4
+
+Benchmark the Model with OpenVINO's Benchmark_App
```
Comment (adrianboguszewski) [inline recipes/intelligent_queue_management/README.md:109]:
```suggestion
Benchmark the Model with OpenVINO's `benchmark_app`.
```

File: recipes/intelligent_queue_management/README.md [hunk 7]
```diff
@@ -104,4 +104,46 @@ _NOTE: Alternatively, you can run all steps with the following command:_
...
+| GPU | N/A | N/A | 1x Intel® Arc A770™ <br>16GB, 512 EU | 1x Intel® Iris® <br>Xe Graphics | 1x Intel® Data Center <br>GPU Flex 170 | 1x Intel® Arc A770™ <br>16GB, 512 EU | 1x Intel® Iris® <br>Xe Graphics |
+| Workload: <br>Codec, <br>resolution, <br>frame rate<br> Model, size (HxW), BS | Yolov8m Model<br>– input size [640, 640], batch 1<br> FP16 \| int8 | Yolov8m Model<br>– input size [640, 640], batch 1<br> FP16 \| int8 | Yolov8m Model<br>– input size [640, 640], batch 1<br> FP16 \| int8 | Yolov8m Model<br>– input size [640, 640], batch 1<br> FP16 \| int8 | Yolov8m Model<br>– input size [640, 640], batch 1<br> FP16 \| int8 |  Yolov8m Model<br>– input size [640, 640], batch 1<br> FP16 \| int8 | Yolov8m Model<br>– input size [640, 640], batch 1<br> FP16 \| int8 |
+| TDP | 350W | 15W | 45W | 235W | 45W | 45W |
+| Benchmark Date | May 31, 2023 | May 29, 2023 | June 15, 2023 | May 29, 2023 | June 15, 2023 | May 29, 2023 
+| Benchmarked by | Intel Corporation | Intel Corporation | Intel Corporation | Intel Corporation | Intel Corporation | Intel Corporation |
+
+Congratulations! You have successfully set up and run the Intelligent Queue Management application with OpenVINO™.
```
Comment (adrianboguszewski) [inline recipes/intelligent_queue_management/README.md:149]:
```suggestion
Congratulations! You have successfully set up and run the Intelligent Queue Management application with OpenVINO™.

### Appendix

#### Platform Configurations for Performance Benchmarks for YOLOv8m Model

| Type Device | | CPU | | | GPU | |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| System Board | Intel Corporation<br>D50DNP1SBB | AAEON<br>UPN-ADLN01 V1.0<br>220950173 | Intel® Client Systems<br>NUC12SNKi72 | Intel Corporation<br>M50CYP2SBSTD | Intel® Client Systems<br>NUC12SNKi72 | Intel® Client Systems<br>NUC12SNKi72 |
| CPU | Intel(R) Xeon(R) <br>Platinum 8480+ | Intel® Core™ <br>i3-N305 @ 3.80 GHz | 12th Gen Intel® Core™ <br>i7-12700H @ 2.30 GHz | Intel(R) Xeon(R) <br>Gold 6348 CPU @ 2.60GHz | 12th Gen Intel® Core™ <br>i7-12700H @ 2.30 GHz | 12th Gen Intel® Core™ <br>i7-12700H @ 2.30 GHz |
| Sockets / Physical cores | 1 /  56 <br>(112 Threads) | 1 / 8 <br>(8 Threads) | 1 /14 <br>(20 Threads) | 2 / 28 <br>(56 Threads) | 1 /14 <br>(20 Threads) | 1 /14 <br>(20 Threads) |
| HyperThreading / Turbo Setting | Enabled / On | Disabled | Enabled / On | Enabled / On | Enabled / On | Enabled / On |
| Memory | 512 GB DDR4 <br>@ 4800 MHz | 16GB DDR5 <br>@4800 MHz | 64 GB DDR4 <br>@ 3200 MHz | 256 GB DDR4 <br>@ 3200 MHz | 64 GB DDR4 <br>@ 3200 MHz | 64 GB DDR4 <br>@ 3200 MHz |
| OS | Ubuntu 22.04.2 LTS | Ubuntu 22.04.2 LTS | Windows 11 <br>Enterprise v22H2 | Ubuntu 22.04.2 LTS | Windows 11 <br>Enterprise v22H2 | Windows 11 <br>Enterprise v22H2 |
| Kernel | 5.15.0-72-generic | 5.15.0-1028-intel-iotg | 22621.1702 | 5.15.0-57-generic | 22621.1702 | 22621.1702 |
| Software | OpenVINO 2023.0 | OpenVINO 2023.0 | OpenVINO 2023.0 | OpenVINO 2023.0 | OpenVINO 2023.0 | OpenVINO 2023.0 |
| BIOS | Intel Corp. <br>SE5C7411.86B.9525<br>.D13.2302071333 | American Megatrends <br>International, <br>LLC. UNADAM10 | Intel Corp. <br>SNADL357.0053<br>.2022.1102.1218 | Intel Corp. <br>SE5C620.86B.01<br>.01.0007.2210270543 | Intel Corp. <br>SNADL357.0053<br>.2022.1102.1218 | Intel Corp. <br>SNADL357.0053<br>.2022.1102.1218 |
| BIOS Release Date | 02/07/2023 | 12/15/2022 | 11/02/2022 | 10/27/2022 | 11/02/2022 | 11/02/2022 |
| GPU | N/A | N/A | 1x Intel® Arc A770™ <br>16GB, 512 EU | 1x Intel® Iris® <br>Xe Graphics | 1x Intel® Data Center <br>GPU Flex 170 | 1x Intel® Arc A770™ <br>16GB, 512 EU | 1x Intel® Iris® <br>Xe Graphics |
| Workload: <br>Codec, <br>resolution, <br>frame rate<br> Model, size (HxW), BS | Yolov8m Model<br>– input size [640, 640], batch 1<br> FP16 \| int8 | Yolov8m Model<br>– input size [640, 640], batch 1<br> FP16 \| int8 | Yolov8m Model<br>– input size [640, 640], batch 1<br> FP16 \| int8 | Yolov8m Model<br>– input size [640, 640], batch 1<br> FP16 \| int8 | Yolov8m Model<br>– input size [640, 640], batch 1<br> FP16 \| int8 |  Yolov8m Model<br>– input size [640, 640], batch 1<br> FP16 \| int8 | Yolov8m Model<br>– input size [640, 640], batch 1<br> FP16 \| int8 |
| TDP | 350W | 15W | 45W | 235W | 45W | 45W |
| Benchmark Date | May 31, 2023 | May 29, 2023 | June 15, 2023 | May 29, 2023 | June 15, 2023 | May 29, 2023 
| Benchmarked by | Intel Corporation | Intel Corporation | Intel Corporation | Intel Corporation | Intel Corporation | Intel Corporation |
```

File: recipes/intelligent_queue_management/README.md [hunk 8]
```diff
@@ -104,4 +104,46 @@ _NOTE: Alternatively, you can run all steps with the following command:_
...
+Benchmark the Model with OpenVINO's `benchmark_app`.
+
+Benchmarking provides insight into your YOLOv8 model's real-world performance. Performance may vary based on use and configuration.
+
+### Benchmark Results 
+
+![YOLOv8m Benchmark Results](https://github.com/openvinotoolkit/openvino_notebooks/assets/109281183/2d59819e-61b7-4995-bdf3-a6d1090afdd4)
+![](https://github.com/openvinotoolkit/openvino_notebooks/assets/109281183/bed6fc01-f0d4-4f8e-af6a-703182947232)
+
+Benchmarking was performed on an Intel® Xeon® Platinum 8480+ (1 socket, 56 cores) running Ubuntu 22.04.2 LTS. The tests utilized the YOLOv8m model with OpenVINO 2023.0.
```
Comment (adrianboguszewski) [inline recipes/intelligent_queue_management/README.md:118]:
```suggestion
Benchmarking was performed on an Intel® Xeon® Platinum 8480+ (1 socket, 56 cores) running Ubuntu 22.04.2 LTS. The tests utilized the YOLOv8m model with OpenVINO 2023.0. For complete configuration, please check the Appendix section.
```

**Review by adrianboguszewski — APPROVED** [review]

**Pull request merged**